buildSrc/build.gradle.kts

```diff
@@ -4,6 +4,7 @@ plugins {
 
 repositories {
     mavenCentral()
+    // TODO remove before first public release
```

Comment:
Please create an issue for it

Reply:
you mean jira issue or github issue? 
I see that Rafał created github issue and I wonder if it's good that we have tasks in two places.

Reply:
I think cleanup task in JIRA with subtasks will be ok. I think that kinds of tasks we can store in JIRA

Reply:
Closed the issue, created a task in jira.